person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_writer :id
```

Comment:
- [ ] As mentioned by the previous reviewer this is the `attr_reader` and not `attr_writer`, the id should be read and not written 🙃 😉